pdns/dnsdistdist/dnsdist-async.cc

```diff
@@ -31,7 +31,9 @@ AsynchronousHolder::Data::Data(bool failOpen) :
   d_failOpen(failOpen)
 {
   auto [notifier, waiter] = pdns::channel::createNotificationQueue(true);
+  // NOLINTNEXTLINE(cppcoreguidelines-prefer-member-initializer): how I am supposed to do that?
   d_waiter = std::move(waiter);
+  // NOLINTNEXTLINE(cppcoreguidelines-prefer-member-initializer): how I am supposed to do that?
   d_notifier = std::move(notifier);
```

Comment:
I missed those. What is the problem here? i.e. what is clang-tidy complaining about?

Reply:
So I take it `std::move` is not usable in an initializer list?

Reply:
`clang-tidy` tells us to initialize `d_waiter` and `d_notifier` in the member initializer list, which as far as I know it's not possible.

Reply:
`std::move` is, turning a pair into two member variables, I don't think so?

Reply:
> `std::move` is, turning a pair into two member variables, I don't think so?

Right, I would also guess it's not possible.